### PR TITLE
datovka 4.15.6 -> 4.21.1 

### PR DESCRIPTION
**Pull request by ovlach**
###### Description of changes

update: datovka - https://gitlab.nic.cz/datovka/datovka/-/blob/v4.21.1/ChangeLog 
new package: https://gitlab.nic.cz/datovka/libdatovka (datovka now depends on libdatovka)

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -10770,6 +10770,12 @@
     githubId = 15930073;
     name = "Moritz Scheuren";
   };
+  ovlach = {
```
Comment (AndersonTorres) [inline maintainers/maintainer-list.nix:10773]: split this file in a separate commit.

File: pkgs/applications/networking/datovka/default.nix [hunk 2]
```diff
@@ -2,30 +2,31 @@
...
-  buildInputs = [ libisds qmake qtbase qtsvg libxml2 ];
+  buildInputs = [ libdatovka qmake qtbase qtsvg libxml2 qtwebsockets ];
 
   nativeBuildInputs = [ pkg-config ];
 
   meta = with lib; {
     description = "Client application for operating Czech government-provided Databox infomation system";
     homepage = "https://www.datovka.cz/";
-    license = licenses.lgpl3;
+    license = licenses.gpl3;
```
Comment (AndersonTorres) [inline pkgs/applications/networking/datovka/default.nix:29]:
```suggestion
    license = licenses.gpl3Plus;
```

File: pkgs/development/libraries/libdatovka/default.nix [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  configureFlags = [
+    "--with-docbook-xsl-stylesheets=${docbook_xsl}/xml/xsl/docbook"
+  ];
+
+  buildInputs = [ pkg-config autoreconfHook expat gpgme libgcrypt libxml2 libxslt gnutls curl docbook_xsl ];
+
+  meta = with lib; {
+    description = "Client library for accessing SOAP services of Czech government-provided Databox infomation system";
+    homepage = "https://gitlab.nic.cz/datovka/libdatovka";
+    license = licenses.gpl3;
```
Comment (AndersonTorres) [inline pkgs/development/libraries/libdatovka/default.nix:38]:
```suggestion
    license = licenses.gpl3Plus;
```

**Review by AndersonTorres — CHANGES_REQUESTED** [review]

File: maintainers/maintainer-list.nix (hunk 1 above)
Comment (ovlach) [reply maintainers/maintainer-list.nix:10773]: commit was splitted

File: pkgs/applications/networking/datovka/default.nix (hunk 2 above)
Comment (ovlach) [reply pkgs/applications/networking/datovka/default.nix]: changed

File: pkgs/development/libraries/libdatovka/default.nix (hunk 3 above)
Comment (ovlach) [reply pkgs/development/libraries/libdatovka/default.nix]: changed

**Review by AndersonTorres — APPROVED** [review]
